common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -86,13 +86,6 @@
                     throw new Error("invalid mode: " + this.mode);
                 }
                 this.readOnly = $(".discussion-module").data('read-only');
-                this.model.collection.on("reset", function(collection) {
-                    var id;
-                    id = self.model.get("id");
-                    if (collection.get(id)) {
-                        self.model = collection.get(id);
-                    }
-                });
```

Comment:
@Kelketek one more question: this it was deleted in the same comment that added "retrieveSingleThread" - any chance they are related?